package.json

```diff
@@ -50,7 +51,21 @@
     "**/webdriverio": "5.12.1"
   },
   "beachball": {
-    "gitTags": false
+    "gitTags": false,
+    "groups": [
+      {
+        "name": "React Native Windows Packages",
+        "include": [
+          "packages/@react-native-windows/*",
+          "vnext"
+        ],
+        "disallowedChangeTypes": [
+          "major",
+          "minor",
+          "patch"
```

Comment:
There's a beachball bug which causes this to infinitely recurse right now https://github.com/microsoft/beachball/issues/364